lib/cosmos/packets/structure.rb

```diff
@@ -66,6 +66,20 @@ def defined?
       @sorted_items.length > 0
     end
 
+    # Rename an existing item
+    #
+    # @param item_name [String] Name of the currently defined item
+    # @param new_item_name [String] New name for the item
+    def rename_item(item_name, new_item_name)
```

Comment:
This was previously happening in the macro parsing but since it was messing with internal Structure/Packet state I felt it should be moved into a method here.